monitors.go

```diff
@@ -43,6 +44,14 @@ type Monitor struct {
 	Options Options  `json:"options,omitempty"`
 }
 
+// creator is the creator of the monitor
+type creator struct {
```

Comment:
Can you maybe expose the `creator` struct by writing it with a capital `C`? Otherwise, it's not possible to construct `Monitor`s with `Creator`s outside of the `datadog` package.